keps/sig-apps/3998-job-success-completion-policy/README.md

```diff
@@ -398,214 +398,6 @@ To avoid the above conflicts, terminating policies are evaluated the first befor
 This means that the terminating policies are respected rather than the successPolicies,
```

Comment:
Nit: the `## Release Signoff Checklist` should be updated such that all required bits are checked as needed.

Reply:
I updated the section. If I am missing something there, please let me know.
Thanks!